lib/popup.ts

```diff
@@ -0,0 +1,11 @@
+import { expect, Page } from '@playwright/test';
+
+export const closePopUp = async function (page: Page) {
+  const selector = '[aria-label=close-notification]';
+  try {
+    await expect(page.locator(selector)).toHaveCount(0);
```

Comment:
should we add a shorter timeout? this is otherwise going to take 30 seconds per element ...